app/views/invitation_mailer/new_invitation.html.slim

```diff
@@ -0,0 +1,4 @@
+= "You are invited to #{@organization.name} as #{@invitation.role}"
```

Comment:
@Gataniel add html escaping `"You are invited to #{h @organization.name} as #{h @invitation.role}"`